ecommerce/templates/ecommerce/review_detail.html

```diff
@@ -0,0 +1,160 @@
+              {% for i in "123" %}
+              <i class="fas fa-star" style="color: orange;"></i>
+              {% endfor %}
+              {% for i in "12" %}
+              <i class="fa fa-star-o"></i>
+              {% endfor %}
+            </div>
+            {% elif rv.rate == 2 %}
+            <div style="padding-bottom: 10px;">
+              {% for i in "12" %}
+              <i class="fas fa-star" style="color: orange;"></i>
+              {% endfor %}
+              {% for i in "123" %}
+              <i class="fa fa-star-o"></i>
+              {% endfor %}
+            </div>
+            {% elif rv.rate == 1 %}
+            <div style="padding-bottom: 10px;">
+              <i class="fas fa-star" style="color: orange;"></i>
+              {% for i in "1234" %}
+              <i class="fa fa-star-o"></i>
+              {% endfor %}
+            </div>
+            {% endif %}
+            <h5 class="h5 pb-2">{{ rv.title }}</h3>
+            <p class="fs-md mb-2">
+              {{ rv.content }}
+            </p>
+            {% if comments %}
+                {% for cmt in comments %}
+                <hr class="mt-4 mb-3">
+                <table id="comments">
+                <tbody>
+                  <div class="product-review pb-4">
+                      <div class="d-flex justify-content-between">
+                          <div class="d-flex align-items-center me-4 pe-2">
+                              <div style="padding-right: 20px;">
+                                  <i class="far fa-comment-dots"></i>
+                              </div>
+                              <img class ="rounded-circle" width="50" src="\media\profile_pics\avatar.png" alt="User">
+                              <div class="ps-3" style="padding-left: 20px;">
+                                  <h6 class="fs-sm mb-0">{{ cmt.user.username }}</h6>
+                                  <span class="fs-ms text-muted">{{ cmt.created }}</span>
+                              </div>
+                          </div>
+                          <div class="p-2"><i class="fas fa-reply"></i></div>
+                      </div>
+                      
+                      <div class="ps-3" style="padding-left: 105px;">
+                          <p class="fs-md mb-2">
+                              {{ cmt.comment }}
+                          </p>
+                      </div>
+                  </div>
+                </tbody>
+                </table>
+                {% endfor %}
+            {% else %}
+            <hr class="mt-4 mb-3">
+            <p class="fs-md mb-2">
+                There are no comments for this review yet.
+              </p>
+            {% endif%}
+        </div>
+        {% endfor %}
+    </div>
+</section>
+{% endblock %}
+
+{% block jscode %}
+  <script>
+    $(document).ready(function () {
+      // catch the form's submit event
+      $('#commentForm').submit(function () {
+          // create an AJAX call
```

Comment:
```suggestion
          // create an AJAX call
          debugger;
```
debug xem xử lý lấy data, type và url đã đúng chưa em nha
$(this).serialize() ....